code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -133,8 +133,8 @@
 	return TRUE
 
 /obj/machinery/portable_atmospherics/attackby(obj/item/W, mob/user, params)
-	. = ..()
 	if(!istype(W, /obj/item/tank))
+		. = ..()
```

Comment:
this makes no sense you set return value to ..() and then return false

Reply:
it works though and I see no conflicts,
does `. = ..()` actually return? I was not sure

Reply:
oh fuck he's right

Reply:
no it doesnt return, it sets the default return value for the proc to parent

Reply:
kk fixing rn

Reply:
. is the return value of the proc, what is returned if you do return
you set the return value but then immediatelly did a return FALSE so theres no purpose at all in setting it